http/e2e/e2e.go

```diff
@@ -254,14 +254,19 @@ func Run(cfg Config) error {
 			resp, err := client.Do(req)
 			fmt.Printf("[Wait] Waiting for %s. Timeout: %ds\n", healthCheck.url, timeout)
 			if err == nil {
+				io.Copy(io.Discard, resp.Body)
```

Comment:
```suggestion
				_, err = io.Copy(io.Discard, res.Body)
				if err != nil {
					return err
				}
```
Just like this to make the lint happy? Unsure when it might ever be triggered